generator/list.tmpl

```diff
@@ -1,5 +1,15 @@
 {{- template "header" }}
-# SIGs and Working Groups
+
+---
+title: Community Groups
+linkTitle: Community Groups
+description:  |
+  A list of our community groups: Special Interest Groups, Working Groups, User Groups and Committees.
+weight: 99
+type: docs
+aliases: [ "/groups", "/sigs" ]
+slug: community-groups
+---
```

Comment:
This will actually need to go above the template header for it to be recognized >_<